### ISSUE TITLE
OSError: Model name 'facebook/bart-large-cnn' was not found in tokenizers model name list

### ISSUE THREAD

**Issue by WangHexie** [huggingface/transformers#5579]
# 🐛 Bug

## Information

Model I am using (Bart):**facebook/bart-large-cnn**

Language I am using the model on English

The problem arises when using:
* [x] the official example scripts: summarization 

## To reproduce

run the code below
```python
from transformers import BartTokenizer, BartForConditionalGeneration, BartConfig
tokenizer = BartTokenizer.from_pretrained('facebook/bart-large-cnn')
```

## Expected behavior
load the model

## Error infomation

```
---------------------------------------------------------------------------
OSError                                   Traceback (most recent call last)
<ipython-input-5-3a96f15c5285> in <module>
----> 1 tokenizer = BartTokenizer.from_pretrained("facebook/bart-large-cnn")

~/.conda/envs/hug/lib/python3.7/site-packages/transformers/tokenization_utils_base.py in from_pretrained(cls, *inputs, **kwargs)
   1138 
   1139         """
-> 1140         return cls._from_pretrained(*inputs, **kwargs)
   1141 
   1142     @classmethod

~/.conda/envs/hug/lib/python3.7/site-packages/transformers/tokenization_utils_base.py in _from_pretrained(cls, pretrained_model_name_or_path, *init_inputs, **kwargs)
   1244                     ", ".join(s3_models),
   1245                     pretrained_model_name_or_path,
-> 1246                     list(cls.vocab_files_names.values()),
   1247                 )
   1248             )

OSError: Model name 'facebook/bart-large-cnn' was not found in tokenizers model name list (facebook/bart-base, facebook/bart-large, facebook/bart-large-mnli, facebook/bart-large-cnn, facebook/bart-large-xsum, yjernite/bart_eli5). We assumed 'facebook/bart-large-cnn' was a path, a model identifier, or url to a directory containing vocabulary files named ['vocab.json', 'merges.txt'] but couldn't find such vocabulary files at this path or url.
```

## Environment info

     
- `transformers` version: 3.0.2
- Platform: centos
- Python version: python 3.7.0
- PyTorch version (GPU?): pytorch   1.5.0           cpu_py37hd91cbb3_0 
- Tensorflow version (GPU?): tensorflow-gpu            2.0.0
- Using GPU in script?: No
- Using distributed or parallel set-up in script?: no

**Comment by chris1899** [comment]
Did you get this solved? I cannot load in t5-large.

**Comment by WangHexie** [comment]
> Did you get this solved? I cannot load in t5-large.

solved simply by reactivating the environment and restarting the program. I installed old version of transformers, after upgrading transformers, I forgot to deactivate the environment.
